api/v1beta1/neutronapi_webhook.go

```diff
@@ -69,6 +70,8 @@ func (spec *NeutronAPISpec) Default() {
 	// only container image validations go here
 	if spec.ContainerImage == "" {
 		spec.ContainerImage = neutronAPIDefaults.ContainerImageURL
+	if spec.NeutronAPIRouteTimeout == "" {
```

Comment:
You need to put this validation in the *core* validations block below for it to apply via the OpenStackControlplane CR.

Reply:
I think this is not longer needed since I removed the spec.NeutronAPIRouteTimeout value. Thanks for the input!